prox_config_backup.sh

```diff
@@ -91,16 +93,21 @@ function copyfilesystem {
     tar --warning='no-file-ignored' -cvPf "$_filename1" /etc/.
     tar --warning='no-file-ignored' -cvPf "$_filename2" /var/lib/pve-cluster/.
     tar --warning='no-file-ignored' -cvPf "$_filename3" /root/.
+    tar --warning='no-file-ignored' -cvPf "$_filename4" /var/spool/cron/.
```

Comment:
Cron Jobs are configured under /etc/cron.* or /etc/pve/*. You suggest to backup this folder in case someone configured `crontabs`, correct?

Reply:
Yep! I've found `crontab -e` is much more practical to configure cron jobs than different files in /etc/cron.* (at least for me specifically). 
I rather prefer to have just one big file with all cron jobs easily visible at once, instead of having to look at different files to find the one I need to change/add, especially when working only from the command line, like I usually do.

Also, backing up /var/spool/cron/ will backup all users crontabs, not only root ones (in case there are more users in the server)!

Reply:
okay, thanks for explaining. Just want to avoid backup up stuff that noone actually uses.